tsconfig.production.json

```diff
@@ -3,8 +3,9 @@
     "allowSyntheticDefaultImports": true,
     "alwaysStrict": true,
     "charset": "utf-8",
-    "declaration": true,
-    "declarationMap": true,
+    "declaration": false,
+    "esModuleInterop": true,
+    "declarationMap": false,
```

Comment:
3x why?

Reply:
1. Eslint plugins only work with  `.js` files, `declaration` generates `d.ts` files.
1. was removed in https://github.com/ridedott/eslint-plugin/commit/0d1b41b2e6fa7c18ddca54619c872d55e5350a02 .
1. Since there are no declaration files it does not need to be enabled